aries_cloudagent/anoncreds/routes.py

```diff
@@ -126,11 +127,24 @@ class SchemaPostOptionSchema(OpenAPISchema):
     """Parameters and validators for schema options."""
 
     endorser_connection_id = fields.Str(
-        description="Connection identifier (optional) (this is an example)",
+        description="""
+            Connection identifier (optional) (this is an example)
+            You can set this is you know the endorsers connection id you want to use.
+            If not specified then the agent will attempt to find an endorser connection.
+        """,
         required=False,
         example=UUIDFour.EXAMPLE,
     )
 
+    create_transaction_for_endorser = fields.Bool(
+        description="""
+            Create transaction for endorser (optional, default false). 
+            Use this for agents who don't specify an author role but want to 
+            create a transaction for an endorser to sign.""",
+        required=False,
+        example=False,
+    )
+
```

Comment:
I think this is a good use of options. However, I wonder if the first option might be better handled by having an established default endorser (kind of like how we set our public DID and that persists across calls). And I'm honestly not sure what the second one means; is this perhaps a case where we could simplify the interface by sticking to a reasonable default? In what circumstances would we set this value to True?